templates/case-study/ubuntu-pro-support-for-games-developer.html

```diff
@@ -0,0 +1,176 @@
+{% extends "case-study/base_case-study.html" %}
+
+{% from "macros/_macros-hero.jinja" import hero %}
+{% from "macros/_macros-highlights.jinja" import highlights %}
+{% from "macros/_macros-text.jinja" import text %}
+{% from "macros/_macros-quote.jinja" import quote_wrapper %}
+{% from "macros/_macros-image.jinja" import image_wrapper %}
+{% from "macros/_macros-cta.jinja" import cta %}
+
+{% block title %}
+    Ubuntu Pro helps large games publisher secure code dependencies and avoid costly migration
+{% endblock %}
+
+{% block meta_description %}
+    Securing open source dependencies with Ubuntu Pro &mdash; legacy PHP support saves a game developer time and effort, while they plan a migration path.
+{% endblock %}
+
+{% block meta_company_name %}Ubuntu pro for games developer{% endblock %}
```

Comment:
```suggestion
{% block meta_company_name %}Ubuntu Pro for Games Developer{% endblock %}
```

Reply:
The company name isn't on the copy docs only on design, I've added it to the metadata table